webhooks-extension/src/components/WebhookDisplayTable/WebhookDisplayTable.js

```diff
@@ -191,28 +342,36 @@ export class WebhookDisplayTable extends Component {
               )}
               />
           </div>
+
+          <div className="modal">
+            <Modal open={this.state.showDeleteDialog}
+              className=".delete-modal"
```

Comment:
The . is usually the prefix in the css for a className .... i.e you would have delete-modal as the className and then .delete-modal in the css .... seems odd to have a className starting with a .